README.md

```diff
@@ -59,6 +59,31 @@ A key-map of data related to the received payload in the form of:
 1. Resolve with a parsed hook object
 2. Reject if not able to parse hook
 
+### getCheckoutCommand
+Required parameters:
+
+| Parameter        | Type  | Required |  Description |
+| :-------------   | :---- | :--------| :---- |
+| config        | Object | Yes | Configuration Object |
+| config.branch | String | Yes | Pipeline branch |
+| config.host | String | Yes | Scm host (ex: github.com) |
+| config.org | String | Yes | Scm org (ex: screwdriver-cd) |
+| config.prRef | String | No | PR reference (can be a PR branch or reference) |
+| config.repo | String | Yes | Scm repo (ex: guide) |
+| config.sha | String | Yes | Scm sha |
+
+#### Expected Outcome
+Decorated url in the form of:
+```js
+{
+    command: 'git clone https://github.com/screwdriver-cd/guide'
```

Comment:
Should this give back a step name so that it will be a step object?

If not, we should just make this return a string, which is the command to run.